dimagi/pages/templates/sections/commcare/commcare_ict4d/tools.html

```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="tools align-center border-top pad-bottom-line"
+         id="tools">
+  <div class="content pad-vert-medium">
+    <div class="content-width-large">
+      <h2 class="blue gap-top-line">
+        {% blocktrans %}
+          Give your teams the tool <br class="hide-until-medium" />
+          they’ve been waiting for
+        {% endblocktrans %}
+      </h2>
+      <p class="lead-text">
+        {% blocktrans %}
+        Your teams have been waiting for a tool that actually aligns with 
+        their needs. Meet the ambitions <br class="hide-until-medium" /> of your Program and M&E teams with 
+        custom-built CommCare applications and powerful data tools.
+        {% endblocktrans %}
+      </p>
+	</div>
```

Comment:
indentation here needs to be fixed, not clear on first glance what this div belongs to